.travis.yml

```diff
@@ -6,10 +6,10 @@ node_js:
 sudo: true
 
 env:
-  - CXX=g++-4.8 NODE_ENV=testing
+  - CXX=g++-4.8 NODE_ENV=testing DEPLOYED=0
```

Comment:
This is necessary for this PR ?

Reply:
it's not necessary for this PR, but I missed the things of the PR test lanter, bad that